runelite-client/src/main/java/net/runelite/client/plugins/specialcounter/SpecialCounterPlugin.java

```diff
@@ -286,13 +302,14 @@ public void onHitsplatApplied(HitsplatApplied hitsplatApplied)
 		// venge or thralls.
 		if (hitsplatTick == client.getTickCount())
 		{
+			secondToLastSpecHitsplat = lastSpecHitsplat;
 			lastSpecHitsplat = hitsplat;
 		}
 	}
 
 	private void specialAttackHit(SpecialWeapon specialWeapon, Hitsplat hitsplat, NPC target)
```

Comment:
replace the `hitsplat` parameter here with `int hit` and modify the call site to compute the hit